src/controllers/softtakeover.h

```diff
@@ -13,13 +13,11 @@
 
 class ControlObject;
 
-// 3/128 units away from the current is enough to catch fast non-sequential moves
-//  but not cause an audibly noticeable jump.
-
-
 class SoftTakeover {
   public:
-    static const double kDefaultTakeoverThreshold;
+    // 3/128 units away from the current is enough to catch fast non-sequential moves
+    //  but not cause an audibly noticeable jump.
+    static const double kDefaultTakeoverThreshold = 3.0 / 128;
```

Comment:
Wait a sec, it's done with "static const uint SUBSEQUENT_VALUE_OVERRIDE_TIME_MILLIS = 50;" so if that works, this should too, right?